src/web/app/pages-admin/admin-search-page/admin-search-page.component.scss

```diff
@@ -5,3 +5,7 @@
 .margin-left-5px {
   margin-left: 5px;
 }
+
+.course-deleted {
+  color: gray;
+}
```

Comment:
> 1. While not as big a deal since this is not a user facing page, the contrast is a little too low here which affects accessibility (The current contrast ratio of the gray on gray sections is about 3.33:1, which is below WCAG's recommendation of 4.5:1).
> 2. When using a UI framework like bootstrap, try to look for suitable pre-defined css utility classes first before defining our own (e.g. the text-muted class is quite close to this). This makes it a lot easier to keep the UI consistent across the site.

Thanks for the suggestion and I just learned something new. Previously I only considered the contrast between the gray and the normal black color but not the background color. I just tried different colors and #6B6B6B seems to be the one giving exactly 4.5:1. Although it reduces the contrast ratio with the black, the recycle-bin icon should serve its purpose as a differentiation. If it's okay, I'll proceed to use #6B6B6B instead of text-muted. The special color hex shouldn't affect a lot because this seems to be an independent function and css.

Reply:
Updated:
![image](https://user-images.githubusercontent.com/68739475/169342383-fb61ca2d-72c7-42d7-8ccf-3217f6fcca46.png)

Reply:
I preferred option b.

Reply:
![image](https://user-images.githubusercontent.com/68739475/169347495-6349080f-56f3-4849-8064-6a475c8c5bd3.png)
![image](https://user-images.githubusercontent.com/68739475/169347582-85732a5c-ef2a-4202-acc0-74a5533a7eb1.png)